src/test/ui/reify-intrinsic.stderr

```diff
@@ -8,9 +8,9 @@ LL |     let _: unsafe extern "rust-intrinsic" fn(isize) -> usize = std::mem::tr
    |                                                                help: use parentheses to call this function: `std::mem::transmute(...)`
    |
    = note: expected type `unsafe extern "rust-intrinsic" fn(isize) -> usize`
-              found type `unsafe extern "rust-intrinsic" fn(_) -> _ {std::intrinsics::transmute::<_, _>}`
+              found type `unsafe extern "rust-intrinsic" fn(_) -> _ {std::mem::transmute::<_, _>}`
```

Comment:
I'm not really sure why this PR causes this error message to get changed (although it's definitely an improvement)

Reply:
I recall @alexreg getting similar diff changes on a PR. cc @oli-obk 

(Do you happen to do development on a macOS?)

Reply:
No - I'm using Linux.